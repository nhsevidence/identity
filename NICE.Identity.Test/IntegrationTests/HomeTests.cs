using System;
using NICE.Identity.Test.Infrastructure;
using System.Threading.Tasks;
using Xunit;

namespace NICE.Identity.Test.IntegrationTests
{
	public class HomeTests : TestBase
	{
		[Fact]
		public async Task GetHomepage()
		{
			//Arrange
			var client = GetClient();

			//Act
			var response = await client.GetAsync("/");
			response.EnsureSuccessStatusCode();
			var responseString = await response.Content.ReadAsStringAsync();

			//Assert
			responseString.ShouldMatchApproved(new Func<string, string>[] { Scrubbers.ScrubCookieString });
		}

	}
}
