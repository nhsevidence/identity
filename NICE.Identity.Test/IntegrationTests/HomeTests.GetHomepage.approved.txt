
<!DOCTYPE html>
  <html>
    <head>
      <meta charset="utf-8" />
      <meta name="viewport" content="width=device-width, initial-scale=1.0" />
      <title>
        Home - NICE
      </title>
      <link rel="stylesheet" href="/css/style.css?v=versionScrubbedByScrubVersion" />
      </head>
      <body>
        <div class="container">
          <p class="phase-banner">
            <span class="phase-banner__tag">
              <span class="tag tag--impact tag--alpha">
                Alpha
              </span>
            </span>
            <span class="phase-banner__label">
              
				NICE Identity is in development. This means it is not feature complete and there may be issues.
			
            </span>
          </p>
        </div>
        <div class="container body-content">
          <div class="page-header">
            <h1 class="page-header__heading">
              
		NICE Identity
	
            </h1>
            <p class="page-header__lead">
              
		Signed in: False
	
            </p>
          </div>
          <hr />
          <footer>
            <p>
              &copy; 2019 - NICE
            </p>
          </footer>
        </div>
        <script src="https://ajax.aspnetcdn.com/ajax/jQuery/jquery-1.12.4.min.js" crossorigin="anonymous" integrity="sha384-nvAa0+6Qg9clwYCGGPpDQLVpLNn0fRaROjHqs13t4Ggj3Ez50XnGQqc/r8MhnRDZ">
        </script>
        <script>
          (window.jQuery||document.write("\u003Cscript src=\u0022\/lib\/jquery\/dist\/jquery.min.js\u0022 crossorigin=\u0022anonymous\u0022 integrity=\u0022sha384-nvAa0\u002B6Qg9clwYCGGPpDQLVpLNn0fRaROjHqs13t4Ggj3Ez50XnGQqc\/r8MhnRDZ\u0022\u003E\u003C\/script\u003E"));
        </script>
        <script src="/js/main.js?v=versionScrubbedByScrubVersion">
        </script>
      </body>
    </html>